lib/r7_insight/host/connection.rb

```diff
@@ -189,7 +189,8 @@ def open_connection
             ssl_context.cert_store = cert_store
 
             ssl_context.min_version = OpenSSL::SSL::TLS1_1_VERSION
-            ssl_context.max_version = OpenSSL::SSL::TLS1_3_VERSION
+            # For older versions of openssl (prior to 1.1.1) missing support for TLSv1.3
+            ssl_context.max_version = defined?(OpenSSL::SSL::TLS1_3_VERSION) ? OpenSSL::SSL::TLS1_3_VERSION : OpenSSL::SSL::TLS1_2_VERSION
```

Comment:
just to avoid long lines and multi-line ternaries
```suggestion
            ssl_context.max_version = if defined?(OpenSSL::SSL::TLS1_3_VERSION)
                                        OpenSSL::SSL::TLS1_3_VERSION
                                      else
                                        OpenSSL::SSL::TLS1_2_VERSION
                                      end
```

Reply:
Also if you could run `make bump-patch` to bump versions

Reply:
Thank you so much for the review and the suggestion.  Really appreciated it.